apmpackage/apm/data_stream/service_metrics/fields/ecs.yml

```diff
@@ -0,0 +1,137 @@
+- external: ecs
```

Comment:
this field set is too liberal. Need to prune this down to only the fields we actually intend to use.